zomes/rea_economic_resource/storage/src/lib.rs

```diff
@@ -130,6 +136,7 @@ impl From<CreationPayload> for EntryData
             current_location: if r.current_location == MaybeUndefined::Undefined { None } else { r.current_location.to_owned().to_option() },
             contained_in: if r.contained_in == MaybeUndefined::Undefined { None } else { r.contained_in.to_owned().to_option() },
             note: if r.note == MaybeUndefined::Undefined { None } else { r.note.clone().into() },
+            primary_accountable: if String::from(e.action.to_owned()) == produce_action.id || String::from(e.action.to_owned()) == raise_action.id { Some(e.receiver) } else { None },
```

Comment:
```suggestion
            primary_accountable: if action_id == produce_action.id || action_id == raise_action.id || action_id == lower_action.id { Some(e.receiver) } else { None },
```

let's move the stringification above, and also add the `lower` action, which can be used with `newInventoriedResource`